patches/patchit.sh

```diff
@@ -1,2 +1,3 @@
 #!/bin/bash
 patch --quiet -p0 -N node_modules/razzle/config/createJestConfig.js < patches/razzle-jest.patch
+git apply --directory=node_modules/formidable/ patches/fixformidable.diff
```

Comment:
Why git and not patch?

Reply:
I don't know the patch magic incantation to apply to multiple files.